src/docfx/lib/markdown/ExtractTitle.cs

```diff
@@ -23,6 +24,10 @@ public static MarkdownPipelineBuilder UseExtractTitle(this MarkdownPipelineBuild
                     Markup.Result.TitleHtml = RenderTitle(h1);
                     document.Remove(h1);
                 }
+                else
+                {
+                    throw Errors.H1NotFound((Document)InclusionContext.File).ToException();
```

Comment:
don't throw, this should be an warning, add it to errors list

Reply:
+1